docs/js-lang-procedures.md

```diff
@@ -0,0 +1,34 @@
+# JS stored function or procedure
+
+After component is installed it becomes possible to create stored function or stored procedure in JS language using the following syntax: 
```

Comment:
After component -> After a component 

to create stored function -> to create a stored function